megamek/src/megamek/common/MiscType.java

```diff
@@ -489,15 +496,10 @@ public double getTonnage(Entity entity, int location) {
                 // be split across 3 instances, since it's spreadable equipment
                 return (0.250 / 3);
             } else {
-                if (hasSubType(S_JETBOOSTER)) {
-                    return entity.hasEngine() ? entity.getEngine().getWeightEngine(entity) / 10.0f : 0.0f;
-                }
                 if (hasSubType(S_SUPERCHARGER)) {
-                    Engine e = entity.getEngine();
-                    if (e == null) {
-                        return 0.0f;
-                    }
-                    return Math.ceil((e.getWeightEngine(entity) / 10.0f) * 2.0f) / 2.0;
+                    // pg 411, TO
+                    // 10% of engine weight
+                    return entity.hasEngine() ? entity.getEngine().getWeightEngine(entity) / 10.0f : 0.0f;
```

Comment:
Per TacOps, p. 344 the weight of the supercharger is rounded up to the nearest half ton for units over 5t and to the nearest kg for units under 5t. I suspect a rules inquiry would say the same is true of the VTOL jet booster but that's not what the rules currently say.